src/App.tsx

```diff
@@ -1,223 +1,142 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
 
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+
+import { NewTodo } from './components/NewTodo';
+import { TodoList } from './components/TodoList';
+import { Filter } from './components/Filter';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Errors } from './types/Errors';
+
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [todoList, setTodolist] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<Filters>(Filters.All);
+  const [showError, setShowError] = useState<Errors>(Errors.None);
+
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const countActivaTodos = todosFromServer.filter(
+    todo => !todo.completed,
+  ).length;
+
+  async function loadingTodos() {
+    if (!user) {
+      return;
+    }
+
+    const todos = await getTodos(user.id);
+
+    try {
+      setTodosFromServer(todos);
+      setTodolist(todos);
+    } catch (error) {
+      throw new Error(`${error}`);
+    }
```

Comment:
There is no point in catching a bug and then throwing it away. If you caught the error, then someone already threw it away. So, either process it, show it to the user, write in logs or something else useful to learn more about the error, or don't catch it at all so that it appears in the project to the final result